### PR TITLE
add note about owners

### PR DESCRIPTION
**Pull request by notwaldorf**
This time in the right repo!
@tjsavage @cdata

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-# Polymer Elements
-## Guide for Contributors
+# Contributing to the Polymer Elements
 
-Polymer Elements are built in the open, and the Polymer authors eagerly encourage any and all forms of community contribution. When contributing, please follow these guidelines:
+🎉First of all, thank you for taking time to contribute! 🎉
+
+Polymer Elements are built in the open, and the Polymer authors eagerly encourage any and all forms of community contribution.
+
+Each element is "owned" by some members of the Polymer team, which you can find in **OWNERS.md** file. They will be your main point of contact, so please cc them when filing an issue or submitting a PR. This will help with triage and making sure your issue or PR is looked at by the right people.
```
Comment (rictic) [inline CONTRIBUTING.md:7]: Would it be helpful to automate this?

**Comment by tjsavage** [comment]
LGTM!

**Comment by cdata** [comment]
LGTM :+1:

**Comment by rictic** [comment]
This is fine by me. When this is merged it'll get pushed out to all of the repos soon after, so only merge when you want that to happen!

File: CONTRIBUTING.md (hunk 1 above)
Comment (notwaldorf) [inline CONTRIBUTING.md:7]: Whoa! Magic. Maybe? Let's lean on the lazy side and say "no for now" and see if that works, and if it doesn't, automate it?

**Comment by notwaldorf** [comment]
@rictic Should I add the `OWNERS.md` file here too, or is that a Different Tedium Task?

**Comment by rictic** [comment]
That's a different task

**Comment by notwaldorf** [comment]
I'm going to close this PR, since I think we're not going down this path anymore, but we can re-open it later if we want

**Pull request closed without merging**